integration/Makefile

```diff
@@ -0,0 +1,2 @@
+test:
```

Comment:
I would suggest using the same pattern that we already use for tests, where GH actions calls the Tracee makefile, which calls into tracee-rules makefile, which knows how to run the tests. some thoughts:
1. this tests tracee-ebpf specifically, it doesn't test "Tracee" so it should be under the tracee-ebpf dir.
2. tracee-ebpf already has a tests directory. I would put `integration` there. (nit is this integration test or e2e test, given the mention scoping of tracee-ebpf)
3. if it's under tracee-ebpf, is there a reason why it needs it's own makefile, or could it be added to the tracee-rule's makefile.

Reply:
Updated https://github.com/aquasecurity/tracee/pull/606/commits/a630a561913db092b0a10602969048469576431b

Reply:
you moved it into tracee-ebpf rather than into tracee-ebpf/test as suggested. don't you think it belongs in the test dir?

Reply:
I'm fine either way. I just took the conventional approach of placing it under `integration/`. One example is Trivy https://github.com/aquasecurity/trivy/tree/main/integration I reworked  as you asked: https://github.com/aquasecurity/tracee/pull/606/commits/efd2dcff53a5da95b8eedca40cd1cf65215170bd